README.MD

```diff
@@ -1,2 +1,6 @@
 ### GIT
-This is the first commit from master branch
\ No newline at end of file
+This is the first commit from master branch
+
+### feature-1
+* Developer is developing the applications
```

Comment:
I think we need to change something over here can you do that or is enough

Reply:
Firstly, we push if there any problem we can update it later